packages/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```diff
@@ -2029,16 +2064,25 @@ describe('plugin-meetings', () => {
                   muted: false,
                   underlyingTrack: webrtcAudioTrack2
                 };
+                const fakeMicrophoneStream = {
+                  on: sinon.stub(),
+                  off: sinon.stub(),
+                  outputStream:{
+                    getTracks: () => {
+                      return [fakeMicrophoneTrack2]
```

Comment:
fakeMicrophoneTrack2 is being used here as a webrtc raw track, so I think it would be better to rename it so that it's clear and also we should update it to reflect the interface of MediaStreamTrack, so for example methods like on/off, setUnmuteAllowed, setPublished etc should be removed from it

Reply:
As mentioned in one of the previous comments, have removed `fakeMicrophoneTrack2` & `fakeMicrophoneTrack` and instead set a sample track with ID since the test's aim is to check if the track from the stream is properly sent to the transcoded meeting `RoapMediaConnection.update`